README.md

```diff
@@ -83,6 +83,7 @@ The following table lists the configurable parameters of the Concourse chart and
 
 | Parameter               | Description                           | Default                                                    |
 | ----------------------- | ----------------------------------    | ---------------------------------------------------------- |
+| `concourse.web.auth.userIDFieldPerConnector` | Define how to display user ID for each authentication connector. | `nil` |
```

Comment:
This could be more clear in the README, but we don't document Concourse binary config options here. These are k8s/helm configs in this table.

```suggestion
```